eng/tools/sdk-suppressions/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "@azure-tools/sdk-suppressions",
+  "private": true,
+  "type": "module",
+  "main": "dist/src/index.js",
+  "version": "1.0.0",
+  "bin": {
+    "get-sdk-suppressions-label": "cmd/sdk-suppressions-label.js"
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "test": "vitest",
+    "test:ci": "vitest run --coverage --reporter=verbose"
+  },
+  "engines": {
+    "node": ">= 18.0.0"
+  },
+  "dependencies": {
+    "ajv": "^6.12.6",
```

```suggestion
    "ajv": "^8.17.1",
```

Use latest if possible.

---

Updated version